sys/universal_address/universal_address.c

```diff
@@ -28,7 +31,11 @@
 /**
  * @brief Maximum number of entries handled
  */
+#ifdef MODULE_FIB
+#define UNIVERSAL_ADDRESS_MAX_ENTRIES (2 * FIB_MAX_FIB_TABLE_ENTRIES)
```

Comment:
just a comment (no need to change something)
Twice the `FIB_MAX_FIB_TABLE_ENTRIES` is safe but only necessary if all addresses (destination and next-hop) used in for the FIB entries are unique.
Multiple equal addresses only occupy a single memory for the address.